src/hotspot/cpu/aarch64/templateTable_aarch64.cpp

```diff
@@ -810,7 +810,7 @@ void TemplateTable::aaload()
   // r0: array
   // r1: index
   index_check(r0, r1); // leaves index in r1, kills rscratch1
-  if (ValueArrayFlatten) {
+  if (FlatArrayFlatten) {
```

Comment:
FlatArrayFlatten is not very explicit.
What about UseFlatArray?

Reply:
Sure.  That's a better name.